### ISSUE TITLE
Windows Logon: NVDA doesn't read alert when entering incorrect password

### ISSUE THREAD

**Issue by chojiro** [nvaccess/nvda#10881]
When entering a password incorrectly at Windows logon screens, NVDA doesn't read the alert that appears. You can either do this when you first boot up your system and Windows logon appears, or by getting to the Windows logon screen with Windows+L after logging on. It doesn't seem to matter which approach I use. This has never worked for me.

### Steps to reproduce:
1. When logging onto the system enter your password incorrectly.
2. You can verify something happened and that you are on the OK button with NVDA commands.

### Actual behavior:
NVDA says "Welcome", but nothing else and does not verify that something actually changed on-screen or that the focus changed to a button.
### Expected behavior:
NVDA should have spoken the text that appears, as well as focus changes.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2019.3.1
#### Windows version:
Windows 10 version 1909
#### Does the issue still occur after restarting your computer?
Yes

**Comment by josephsl** [comment]
Hi, it does read on my system running Version 2004. Thanks.

**Comment by chojiro** [comment]
Not sure what to tell you.

Upgrading to 2004 build 19041.153 and still no luck. Installing NVDA alpha-19797,c8baa820 and that is still the case. Attempted to do this repeatedly and all NVDA ever says is "Welcome", then is completely silent afterwards. I can get it to read that it is incorrect by using NVDA+B to read the entire window, but otherwise NVDA says nothing.

**Comment by josephsl** [comment]
Hi, I’ll take a look today and see if the data matches my hunches (provided that I’m feeling a bit better physically). Thanks.

**Comment by josephsl** [comment]
Hi, after you hear “Welcome”, I assume NVDA isn’t saying “dialog”? If yes, then it might be that NVDA isn’t recognizing the “incorrect password” window as a proper dialog like it should – that’s how LogonUI (the app responsible for gathering credentials) informs you about incorrect passwords. I’m interested to find out if others are also experiencing this problem. Thanks.

**Comment by chojiro** [comment]
That's correct. NVDA recognizes the initial screen as a dialog, though. I  hear "Windows sign-in dialog" when I first boot or when hitting Windows+L and NVDA starts, but NVDA just says "Welcome" after entering the wrong password.

**Comment by josephsl** [comment]
Hi, hmmm, that’s odd indeed. Looks like focus events are not fired right. CC @MichaelDCurran

**Comment by Adriani90** [comment]
@chojiro are you still having this issue in NVDA 2023.2?

**Comment by chojiro** [comment]
No. I am no longer seeing this. I forgot about it when it started working again so forgot to close.

**Comment by Adriani90** [comment]
Thanks for the quick reply. Closing as works for me.
